---
title: Investigation of receptor interacting protein (RIP3)-dependent protein phosphorylation
  by quantitative phosphoproteomics
date: '2012-12-01'
draft: false
publishDate: '2023-08-03T08:27:23.558220Z'
authors:
- Xiurong Wu
- Lili Tian
- Jie Li
- Yingying Zhang
- Victor Han
- Yuanyue Li
- Xiaozheng Xu
- Hanjie Li
- Xi Chen
- Jinan Chen
- Wenhai Jin
- Yongming Xie
- Jiahuai Han
- Chuan-Qi Zhong
publication_types:
- '2'
abstract: Receptor interacting protein 3 (RIP3) is a protein kinase that plays a key
  role in programmed necrosis. Despite the importance of RIP3-dependent necrosis in
  many pathological processes, current knowledge on the function of RIP3 is very limited.
  Here we present the results of a proteome-wide analysis of RIP3-regulated phosphorylation
  sites using cells from wildtype (RIP3(+/+)) and RIP3 knockout (RIP3(-/-)) mice.
  Because the activation of RIP3 requires stimulation by certain extracellular stimuli
  such as ligands of death receptors or Toll-like receptors, we compared the phosphorylation
  sites of lipopolysaccharide (LPS)-treated peritoneal macrophages from RIP3(+/+)
  and RIP3(-/-) mice and the phosphorylation sites of tumor necrosis factor (TNF)-treated
  RIP3(+/+) and RIP3(-/-) mouse embryonic fibroblast (MEF) cells. Stable isotope labeling
  by amino acids in cell culture and spike-in stable isotope labeling by amino acids
  in cell culture were used in the analyses of the MEFs and macrophages, respectively.
  Proteomic analyses using stable isotope labeling by amino acids in cell culture
  coupled with immobilized metal affinity chromatography-hydrophilic interaction liquid
  chromatography fractionation and nanoLC MS/MS identified 14,057 phosphopeptides
  in 4306 proteins from the macrophages and 4732 phosphopeptides in 1785 proteins
  from the MEFs. Analysis of amino acid sequence motifs among the phosphopeptides
  identified a potential motif of RIP3 phosphorylation. Among the phosphopeptides
  identified, 73 were found exclusively in RIP3(+/+) macrophages, 121 were detected
  exclusively from RIP3(+/+) MEFs, 286 phosphopeptides were induced more in RIP3(+/+)
  macrophages than in RIP3(-/-) macrophages and 26 phosphopeptides had higher induction
  in RIP3(+/+) MEFs than in RIP3(-/-) cells. Many of the RIP3 regulated phosphoproteins
  from the macrophages and MEF cells are functionally associated with the cell cycle;
  the rest, however, appear to have diverse functions in that a number of metabolism
  related proteins were phosphorylated in macrophages and development related phosphoproteins
  were induced in MEFs. The results of our phosphoproteomic analysis suggest that
  RIP3 might function beyond necrosis and that cell type specific function of RIP3
  exists.
featured: false
publication: '*Molecular & cellular proteomics*'
tags:
- Proteome
- Amino Acid Motifs
- Amino Acid Sequence
- Amino Acids
- Animals
- Cell Cycle
- Cell Line
- Chromatography
- Affinity
- Chromatography
- Liquid
- Fibroblasts
- Isotope Labeling
- Lipopolysaccharides
- Macrophages
- Peritoneal
- Mice
- Mice
- Knockout
- Necrosis
- Phosphopeptides
- Phosphorylation
- Proteomics
- Receptor-Interacting Protein Serine-Threonine Kinases
- Sequence Analysis
- Protein
- Signal Transduction
- Staining and Labeling
- Tumor Necrosis Factor-alpha
doi: 10.1074/mcp.M112.019091
---

